common/SmokeTests/SmokeTest/SmokeTest.csproj

```diff
@@ -1,8 +1,9 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFrameworks>netcoreapp2.1;net461</TargetFrameworks>
+    <NoWarn>$(NoWarn);NU1605</NoWarn>
```

Comment:
The EventHUbs issue has been fixed see https://github.com/Azure/azure-sdk-for-net/pull/11209. So I wonder if we should still do this or not. It may help us identify issues sooner but as we found it could also have false positives. I kind of want to skip out on this fix for as long as possible. Perhaps add it commented out now with a comment that says we might need to disable this in some cases if libraries end up depending on a prerelease instead of dev.